etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -609,7 +600,7 @@ f5_bigip_lbaas_device_driver = f5_openstack_agent.lbaasv2.drivers.bigip.icontrol
 # If a single IP address is used and the HA model 
 # is not standalone, all devices in the sync failover
 # device group for the hostname specified must have 
-# there management IP address reachable to the agent.
+# their management IP address reachable to the agent.
 # If order to access devices' iControl® interfaces via
 # SelfIPs, you should specify them as a comma 
```

Comment:
We need to make sure we're referencing selfips consistently throughout. So far, I've seen 'selfip', 'SelfIP' and 'Self IP'.